sonar-jmeter-plugin/pom.xml

```diff
@@ -18,7 +18,7 @@
   <properties>
     <sonar.pluginClass>org.sonar.plugins.jmeter.JMeterPlugin</sonar.pluginClass>
     <sonar.pluginName>JMeter</sonar.pluginName>
-    <sonar.version>3.7</sonar.version>
+    <sonar.version>5.1</sonar.version>
```

Comment:
But then it's not v5 compatible, isn't it? Is there a possibility to support both versions?